automation/taskcluster/decision_task.py

```diff
@@ -153,20 +153,21 @@ def nightly_to_production_app(is_staging, version_name):
     # keep updating this "backwards-compatible" nightly for a while yet
     variant = get_variant('fenixNightlyLegacy', 'geckoNightly')
     taskcluster_apk_paths = variant.upstream_artifacts()
+    channel = 'production'  # We're publishing to the "production" app on the "nightly" track
```

Comment:
We need to push to the "production" channel of Google Play. But I think we do not actually build the `production` build type here.

From the code in `craft_assemble_release_task()` I think it expects different values now after we changed the build types:
* The secret is under `nightlyLegacy` (Maybe that should be renamed now)
* The assemble task should be something like `geckoNightlyFenixNightlyLegacy`

🤔

Reply:
Indeed, thanks -- I think my updated PR fixes this...